hack/deploy/controlplane/external-etcd-patch.yaml

```diff
@@ -9,6 +9,7 @@ spec:
       - name: controlplane
         args:
           - "/multicluster-controlplane"
+          - "server"
```

Comment:
need to specify the `--self-management`

Reply:
yes, I want to make this later, I want to have a pr to enhance the deploy script

Reply:
You may do it once https://github.com/open-cluster-management-io/multicluster-controlplane/pull/37 is merged. There are lots of changes to simplify the deploy scripts.